docs/en/pid-eaa-issuance.rst

```diff
@@ -1309,6 +1494,78 @@ In the following table are listed HTTP Status Codes and related error codes that
       - `-`
       - The PID/(Q)EAA Issuer cannot fulfill the request within the defined time interval.
 
+Deferred Endpoint
+-----------------
+
+PID/(Q)EAA Providers MAY support the *Deferred Endpoint* aiming to satify the cases where an immediate issuance might be not possible, due to errors during the communication between the PID/(Q)EAA Provider and the Authentic Source (for example the Authentic Source is temporarily unavailable, etc.) or due to administrative or technical processes.
+
+In the case where the Authentic Source and the PID/(Q)EAA Provider are both enabled to use *PDND*, what is described in Section :ref:`Authentic Sources` MUST apply.
+
+
+The following requirements apply:
+
+ 1. The Deferred Credential request MAY also happen several days after the initial Credential request. 
+ 2. The User MUST be informed that the Credential is available and ready to be issued.
+ 3. The Wallet Provider MUST NOT be informed about which Credential is available to be issued or which Credential Issuer the User needs to contact. 
+ 4. The Wallet Instance MUST be informed about the amount of time to wait before making a new Credential request.
+ 5. As, in general, an unavailability may be an unexpected event, the PID/(Q)EAA Provider MUST be able to switch on the fly between a *immediate* and an *deferred* flow. This decision MUST be taken after the authorization step.
+
+
+If PID/(Q)EAA Providers, supporting this flow, are not able to immediately issue a requested Credential, they MUST provide the Wallet Instance with an HTTP Credential Response cointaining the amount of time to wait before making a new Credential request and an identifier of the deferred issuance transaction (*transaction_id*). The HTTP status code MUST be *202* (see Section 15.3.3 of [:rfc:`9110`]). Below a non-normative example is given.
+
+.. code-block:: http
+
+  HTTP/1.1 202 Accepted
+  Content-Type: application/json
+  Cache-Control: no-store
+
+.. literalinclude:: ../../examples/credential-response-deferred.json
+  :language: JSON  
+
+The Wallet Instance MUST use the value given in the *lead_time* parameter to inform the User when the Credential becomes available (e.g. using a local notification triggered by the *lead_time* time value). PID/(Q)EAA Providers MAY send a notification to the User through a communication channel (e.g. email address), if previously provided by the User to the PID/(Q)EAA Provider.
```

Comment:
Could this case also include a message sent in App IO directly from the issuer to the user's CF?

Reply:
If the Issuer uses App IO message system as a communication channel to notify the user this is ok. However, the definition of communication channels is out of scope of this technical specification.